src/client/datascience/jupyter/kernels/types.ts

```diff
@@ -66,6 +67,7 @@ export type RemoteKernelSpecConnectionMetadata = Readonly<{
     interpreter?: undefined;
     kernelSpec: IJupyterKernelSpec;
     kind: 'startUsingRemoteKernelSpec';
+    baseUrl: string;
```

Comment:
This is simpler, also allows us in the future to connect to multiple remotes or even have local + remote at the same time